praw/models/util.py

```diff
@@ -97,6 +97,8 @@ def stream_generator(function, pause_after=None, skip_existing=False,
 
     :param attribute_name: The field to use as an id (default: "fullname").
 
+    Additional arguments will be passed to the function.
```

Comment:
To improve clarity can this be worded `Additional keyword arguments will be passed to ``function``.`